tt_metal/common/test_common.hpp

```diff
@@ -33,14 +32,14 @@ inline std::string get_soc_description_file(const tt::ARCH &arch, tt::TargetDevi
     if (tt_metal_home.back() != '/') {
         tt_metal_home += "/";
     }
-    if (target_device == tt::TargetDevice::Versim && !use_full_soc_desc) {
+    if (target_device == tt::TargetDevice::VCS){
```

Comment:
this isn't VCS specific right?

Reply:
No, I'll change this to `TargetDevice::SiliconRemote` or something similar

Reply:
changed to `TargetDevice::Simulator`